### ISSUE TITLE
训练出现问题，有人有一样的吗

### ISSUE THREAD

**Issue by xiaomingzhong** [zzh8829/yolov3-tf2#210]
2848/2859 [============================>.] - ETA: 1s - loss: 30.7651 - yolo_output_0_loss: 12.7359 - yolo_output_1_loss: 10.0293 - yolo_output_2_loss: 5.3570
2849/2859 [============================>.] - ETA: 1s - loss: 30.7589 - yolo_output_0_loss: 12.7345 - yolo_output_1_loss: 10.0262 - yolo_output_2_loss: 5.3552
2850/2859 [============================>.] - ETA: 0s - loss: 30.7541 - yolo_output_0_loss: 12.7345 - yolo_output_1_loss: 10.0232 - yolo_output_2_loss: 5.3535
2851/2859 [============================>.] - ETA: 0s - loss: 30.7477 - yolo_output_0_loss: 12.7329 - yolo_output_1_loss: 10.0201 - yolo_output_2_loss: 5.3518
2852/2859 [============================>.] - ETA: 0s - loss: 30.7537 - yolo_output_0_loss: 12.7365 - yolo_output_1_loss: 10.0243 - yolo_output_2_loss: 5.3501
2853/2859 [============================>.] - ETA: 0s - loss: 30.7502 - yolo_output_0_loss: 12.7359 - yolo_output_1_loss: 10.0232 - yolo_output_2_loss: 5.3484
2854/2859 [============================>.] - ETA: 0s - loss: 30.7435 - yolo_output_0_loss: 12.7340 - yolo_output_1_loss: 10.0201 - yolo_output_2_loss: 5.3467
2855/2859 [============================>.] - ETA: 0s - loss: 30.7391 - yolo_output_0_loss: 12.7323 - yolo_output_1_loss: 10.0190 - yolo_output_2_loss: 5.3449
2856/2859 [============================>.] - ETA: 0s - loss: 30.7324 - yolo_output_0_loss: 12.7305 - yolo_output_1_loss: 10.0160 - yolo_output_2_loss: 5.3432
2857/2859 [============================>.] - ETA: 0s - loss: 30.7289 - yolo_output_0_loss: 12.7317 - yolo_output_1_loss: 10.0129 - yolo_output_2_loss: 5.3415
2858/2859 [============================>.] - ETA: 0s - loss: 30.7266 - yolo_output_0_loss: 12.7343 - yolo_output_1_loss: 10.0099 - yolo_output_2_loss: 5.33982020-03-21 23:13:00.264938: W tensorflow/core/common_runtime/base_collective_executor.cc:217] BaseCollectiveExecutor::StartAbort Out of range: End of sequence
	 [[{{node IteratorGetNext}}]]
2020-03-21 23:13:00.265170: W tensorflow/core/common_runtime/base_collective_executor.cc:217] BaseCollectiveExecutor::StartAbort Out of range: End of sequence
	 [[{{node IteratorGetNext}}]]
	 [[IteratorGetNext/_2]]
Epoch 00010: saving model to checkpoints/yolov3_train_10.tf
2859/2859 [==============================] - 393s 137ms/step - loss: 30.7196 - yolo_output_0_loss: 12.7322 - yolo_output_1_loss: 10.0068 - yolo_output_2_loss: 5.3380 - val_loss: 33.1784 - val_yolo_output_0_loss: 14.0400 - val_yolo_output_1_loss: 12.3189 - val_yolo_output_2_loss: 4.2941
Epoch 00010: early stopping

**Comment by zzh8829** [comment]
I don't see any problem here, the end of sequence warning is not an error but rather warning from tensorflow.  "early stopping" means you have reached a point where further training does not improve validation loss.

**Comment by xiaomingzhong** [comment]
@zzh8829 total loss : 30.7266 。

**Comment by kindasweetgal** [comment]
我也遇到了这个错误请问解决了么

**Comment by xiaomingzhong** [comment]
> 我也遇到了这个错误请问解决了么

没有，这个项目没法训练，还是用[这个项目](https://github.com/AlexeyAB/darknet)吧，这个项目我训练好了，训练了50小时左右

**Comment by tsk15535904190** [comment]
tf 检测到过拟合 于是提前终止训练

**Comment by xiaomingzhong** [comment]
这个项目没发训练，关了。

**Comment by tsk15535904190** [comment]
> 这个项目没发训练，关了。

我用这个项目成功训练了火焰识别样本
